code/datums/diseases/advance/advance.dm

```diff
@@ -34,10 +34,39 @@
 	var/oldres
 	// The order goes from easy to cure to hard to cure.
 	var/static/list/advance_cures = 	list(
-																	/datum/reagent/consumable/sodiumchloride, /datum/reagent/consumable/sugar, /datum/reagent/consumable/orangejuice,
-									/datum/reagent/medicine/spaceacillin, /datum/reagent/medicine/salglu_solution, /datum/reagent/consumable/ethanol,
-									/datum/reagent/medicine/leporazine, /datum/reagent/medicine/synaptizine, /datum/reagent/toxin/lipolicide,
-									/datum/reagent/silver, /datum/reagent/gold
+									list(	// level 1

```

Comment:
These really don't need the multiline spacing

Reply:
ok

Reply:
Fails travis without multiline spacing. Considering this particular bit is a /tg/ port, I think it probably actually *does* need the multiline

Reply:
Also: when this was ported by Stormcaster (#420), he made the port actually have even worse multilining, and that was merged